### ISSUE TITLE
Job errors are not logged any longer

### ISSUE THREAD

**Issue by dcoraboeuf** [nemerosa/ontrack#620]
When a job runs with an error, no log entry is created any longer, either in the console or in the log repository.

**Comment by dcoraboeuf** [comment]
It seems to be a regression with the introduction of Prometheus metrics (#583) in `DefaultJobListener#onJobError`, when calling the `meterRegistry.counter` method:

```
java.lang.IllegalArgumentException: Prometheus requires that all meters with the same name have the same set of tag keys. There is already an existing meter containing tag keys []. The meter you are attempting to register has keys [job_category, job_id, job_type].
```

**Comment by dcoraboeuf** [comment]
See micrometer-metrics/micrometer#813

**Comment by dcoraboeuf** [comment]
Available in 3.35.11
